raiden/transfer/mediated_transfer/events.py

```diff
@@ -671,3 +671,31 @@ def from_dict(cls, data: Dict[str, Any]) -> "EventUnexpectedSecretReveal":
         )
 
         return restored
+
+
+class EventRouteFailed(Event):
+    """ Event emitted when a route failed.
+
+    A route fails, when a RefundTransfer reaches the initator.
```

Comment:
Could you please add two little things to this?

- Mention the difference from this to `PaymentSentFailed` and why we added it.
- Mention that this is emitted regardless of the `RefundTransfer` being unlocked or not.